From: ihnp4!amdcad!idi!bene!luke!itkin
Subject: Patch to Pnews to save article
Reply-To: itkin@luke.UUCP (Steven List)
Newsgroups: mod.sources
Approved: john@genrad.UUCP

Mod.sources:  Volume 2, Issue 20
Submitted by: itkin@luke.UUCP (Steven List)


The following is a simple patch to Pnews to save the current article.
It is not perfect (it may save articles when you don't necessarily want
to), but it is better than not saving them at all!
------------------------------------------------------------------------
*** Pnews.old	Thu Jun  6 12:44:33 1985
--- Pnews	Sun Jul 21 11:59:24 1985
***************
*** 397,402
  	state=cleanup
  	;;
      cleanup)
  	$rm -f $tmpart
  	exit
  	;;

--- 397,404 -----
  	state=cleanup
  	;;
      cleanup)
+ 	$cat $tmpart >> ${HOME-$LOGDIR}/Articles
+ 	$echo "Article appended to ${HOME-$LOGDIR}/Articles"
  	$rm -f $tmpart
  	exit
  	;;



